Index: app/controllers/category_helper_controller.rb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/controllers/category_helper_controller.rb	(revision 287bef91b49484aa24074b433d2a9fff6f431973)
+++ app/controllers/category_helper_controller.rb	(revision )
@@ -2,6 +2,7 @@
   include ApplicationHelper
 
   def batch_delete
+    params.permit!
     model = params[:category].constantize
 
     model.where(id: params[:ids].split(',')).update_all(enable: false)
@@ -30,6 +31,7 @@
   end
 
   def create
+    params.permit!
     model = params[:category].constantize
 
     if model == ACategory
@@ -54,6 +56,7 @@
   end
 
   def update
+    params.permit!
     model = params[:category].constantize
 
     @category = model.find(params[:id])
